backend/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tokens(
+  id serial PRIMARY KEY,
+  token_value varchar(255)
+);
+
+CREATE TABLE baskets(
+  id serial PRIMARY KEY,
+  name varchar(255) UNIQUE NOT NULL,
+  token_id bigint NOT NULL REFERENCES tokens(id) ON DELETE CASCADE
+);
+
+CREATE TABLE notifications(
+  id serial PRIMARY KEY,
+  basket_id bigint NOT NULL REFERENCES baskets(id) ON DELETE CASCADE,
```

Comment:
Doing a bit of research it seems like you need to match the data type in both tables, so in this case the `id` in the `baskets` table is an `int` so the `basket_id` should also be `int`, if you want to use `bigint` you'll have to make the other `bigserial`

For reference: https://stackoverflow.com/questions/57461052/bigserial-vs-serial-in-postgresql